src/test/assembly/nvptx-kernel-abi/nvptx-kernel-args-abi-v7.rs

```diff
@@ -0,0 +1,258 @@
+// assembly-output: ptx-linker
+// compile-flags: --crate-type cdylib -C target-cpu=sm_86
+// only-nvptx64
+
+// The following ABI tests are made with nvcc 11.6 does.
+//
+// The PTX ABI stability is tied to major versions of the PTX ISA
+// These tests assume major version 7
+//
+//
+// The following correspondence between types are assumed:
+// u<N> - uint<N>_t
+// i<N> - int<N>_t
+// [T, N] - std::array<T, N>
+// &T - T const*
+// &mut T - T*
+
+// CHECK: .version 7
```

Comment:
Concern: who is on the hook for updating these tests when this check fails? I imagine the most likely outcome will be this test getting `ignore`d just like the others, since this is a T3 target.

If you depend on this target continuing to work right, would you be interested being a contact person for the target?

Reply:
nvptx64 is actually listed as a T2 target. It's good that it's guaranteed to build, hopefully the target will see enough the next months/year to be possible to depend on producing proper builds as well.

I do depend on this target to work right. I am fine with having the responsibility of keeping nvptx64 related things up to date, and happy to be able to contribute to rustc.

Since I'm very new to compiler contributions I will realistically depend on support from more experienced contributors, especially in time sensitive issues. I hope this is acceptable.